### PR TITLE
Update upcoming-api-changes.md

### PR DESCRIPTION
**Pull request by zeiler**

### CODE REVIEW

File: upcoming-api-changes.md [hunk 1]
```diff
@@ -20,8 +20,347 @@ watch our [documentation repo on GitHub](https://github.com/Clarifai/docs).
...
+}
+```
+<br>In order to convert from the new format to something similar the old format where you have a dictionary keyed by `age_appearance`, `gender_appearance` and `multicultural_appearance` you could do something like the following as an example in the Python programming language:<br>
+```
+buckets = {}
+for c in outputs[0].data.regions[0].data:
+  buckets.setdefault(c.vocab_id, [])
+  buckets[c.vocab_id].append(c)
+```
+Similar examples could be made for all other langauge. If you need help in the conversion feel free to reach out to support@clarifai.com.
```
Comment (robertatclarifai) [inline upcoming-api-changes.md]: langauge > languages

Comment (robertatclarifai) [reply upcoming-api-changes.md]: If you need help in the conversion feel free to reach out to support@clarifai.com. 
> If you need help in the conversion, feel free to reach out to support@clarifai.com.

File: upcoming-api-changes.md [hunk 2]
```diff
@@ -20,8 +20,347 @@ watch our [documentation repo on GitHub](https://github.com/Clarifai/docs).
...
...ther model types return. This should only effect users of the Celebrity, Demographics, or custom face recognition models where the `data.face` attributes like `data.face.identity`, `data.face.age_appearance`, `data.face.gender_appearance`, and `data.face.multicultural_appearance` will now be returned in the list of `data.concepts` Concept object. The API will return both for a while during the transition to give you time to update your code away from using the `data.face` objects altogether. We are doing this to simplify the API interface and make it more easily compatible for advanced functionality that is coming soon in workflows! The custom face recognition and celebrity models are a simple change to just access the new `data.concepts` field, but the demographics model is a more fundamental change away from having three distinct lists of concept to a single list. In order to cope with this we have introduced a `vocab_id` field in each `data.concepts` entry that is returned by the demographics model so that you can distinguish `age_appearance`, `gender_appearance` and `multicultural_appearance`.<br><br>For example, the previous demographics model prediction output format was:<br>
```
Comment (robertatclarifai) [inline upcoming-api-changes.md]: In order to cope with this we have introduced a 
-> 
In order to cope with this, we have introduced a

**Comment by zeiler** [comment]
Fixed

**Pull request merged**